src/ast/VarList.ts

```diff
@@ -9,10 +9,10 @@ import {TypeCheckError} from "./symbols/TypeTable";
  */
 export class VarList extends AstNode {
 
-    nameToType: Map<string, string> = new Map();
+    nameTypeMap: Map<string, string> = new Map();
```

Comment:
Thanks for the name change; definitely makes more sense calling it`nameTypeMap`, which is exactly what it is 😄